src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -325,7 +339,7 @@ void VideoDecoder::initializeFilterGraphForStream(
     width = *options.width;
     height = *options.height;
   }
-  std::snprintf(description, sizeof(description), "scale=%d:%d", width, height);
+  std::snprintf(description, sizeof(description), "scale=%d:%d:sws_flags=bilinear", width, height);
```

Comment:
Not a priority now, but we can do this in a safer way with `std::stringstream`; it won't require allocating a static sized buffer.